src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsEnforcementSpec.groovy

```diff
@@ -473,4 +477,28 @@ class PriceFloorsEnforcementSpec extends PriceFloorsBaseSpec {
         assert response.seatbid?.first()?.bid?.collect { it.id } == [bidResponse.seatbid.first().bid.first().id]
         assert response.seatbid.first().bid.collect { it.price } == [floorValue]
     }
+
+    def "PBS shouldn't return error when price floor enforcement service enforce bid request"() {
+        given: "BidRequestWithFloors with Stored Action Response and Floor enabled = false"
```

Comment:
Stored Action Response? Maybe you meant Auction?

Reply:
No, we should add `storedAuctionResponse`